Kickstarter-iOS/AppDelegate.swift

```diff
@@ -422,6 +426,48 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
     let task = session.dataTask(with: url)
     task.resume()
   }
+
+  private func configureRemoteConfig() {
+    let remoteConfig = RemoteConfig.remoteConfig()
+
+    // WARNING: Don't actually do this in production!
+    let settings = RemoteConfigSettings()
+    settings.minimumFetchInterval = 0
+    remoteConfig.configSettings = settings
+
+    AppEnvironment.updateRemoteConfigClient(remoteConfig)
+
+    let appDefaults: [String: Any?] = [
+      RemoteConfigFeature.consentManagementDialogEnabled.rawValue: false,
+      RemoteConfigFeature.facebookLoginInterstitialEnabled.rawValue: false
+    ]
+    AppEnvironment.current.remoteConfigClient?.setDefaults(appDefaults as? [String: NSObject])
+
+    AppEnvironment.current.remoteConfigClient?.activate { _, error in
+      guard let remoteConfigActivationError = error else {
+        print("🔮 Remote Config SDK Successfully Activated")
+
+        // TODO: Eventually replace self.viewModel.inputs.didUpdateOptimizelyClient(optimizelyClient), when we remove `MockOptimizelyClient`
+        return
+      }
+
+      print("🔴 Remote Config SDK Activation Failed with Error: \(remoteConfigActivationError.localizedDescription)")
+
+      Crashlytics.crashlytics().record(error: remoteConfigActivationError)
+
+      self.viewModel.inputs.optimizelyClientConfigurationFailed()
+      // we'll rename this later but keep it this way for now - because notifications are attached to it.
+    }
+
+    AppEnvironment.current.remoteConfigClient?.fetch { _, error in
+      if let remoteConfigFetchError = error {
+        print("🔴 Remote Config SDK Fetch Failed with Error: \(remoteConfigFetchError.localizedDescription)")
```

Comment:
Can remove these lines - because we just need to call `fetch`. no need to error handle or anything..from my reading the firebase instructions it seems like a trigger for the `addOnConfiguUpdateListener`